ablySpec/RealtimeClientChannel.swift

```diff
@@ -552,6 +552,60 @@ class RealtimeClientChannel: QuickSpec {
 
                 }
 
+                // RTL6g
+                context("Identified clients with clientId") {
+
+                    // RTL6g1
+                    context("When publishing a Message with clientId set to null") {
+
+                        // RTL6g1a
+                        pending("should be unnecessary to set clientId of the Message before publishing") {
+                            let options = AblyTests.commonAppSetup()
+                            options.autoConnect = false
+                            let client = ARTRealtime(options: options)
```

Comment:
If that's the case then I apologise, I misread it the first time. Bit confused why no one pulled me up on my mistake though!  

I've tried to review the rest of the code beneath the `options.clientId` at the time of my comment and Github is not allowing me to see that, so it's possible the `message` clientId was being set?

Anyway, setting a clientId is valid in the options, and setting one in the message is not valid, in the context of this test.